packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -2987,6 +2987,60 @@ describe("typespec-client-generator-core: package", () => {
       deepStrictEqual(bodyParam.type, shelfModel);
       deepStrictEqual(bodyParam.correspondingMethodParams, createShelfRequest.type.properties);
     });
+    it("formdata model without body decorator in spread model", async () => {
+      const runnerWithCore = await createSdkTestRunner({
+        librariesToAdd: [AzureCoreTestLibrary],
+        autoUsings: ["Azure.Core", "Azure.Core.Traits"],
+        emitterName: "@azure-tools/typespec-java",
+      });
+      await runnerWithCore.compileWithBuiltInAzureCoreService(`
+      model DocumentTranslateResult {
+        @header
+        contentType: "application/octet-stream";
+        @body
+        document: bytes;
+      }
+
+      model DocumentTranslateContent {
+        @header contentType: "multipart/form-data";
+        /** Document to be translated in the form */
+        document: bytes;
+      
+        /** Glossary / translation memory will be used during translation in the form. */
+        glossary?: bytes[];
+      }
+      
+      #suppress "@azure-tools/typespec-azure-core/byos"
+      @post
+      @route("document:translate")
+      op documentTranslate is RpcOperation<
```

Comment:
can you reduce this to `op foo(...DocumentTranslatecontent): void` and then don't even need the azure core runner

Reply:
it didn't repro without the azure core runner